dunamai/__init__.py

```diff
@@ -155,7 +172,8 @@ def __init__(
         stage: Tuple[str, Optional[int]] = None,
         distance: int = 0,
         commit: str = None,
-        dirty: bool = None
+        dirty: bool = None,
+        local: Optional[str] = None
```

Comment:
I'm a little torn about the name "local". Dunamai generally favors Semantic Versioning names where applicable, so my first thought is "metadata", but that could be confused with the existing use of `--(no-)metadata`. I think I'd like to go with "tagged_metadata".

Reply:
yeah initially i had it as metadata, but changed it to prevent the confusion :D